runtime/command/run/run.go

```diff
@@ -49,56 +52,95 @@ func (c *Command) compileAndStart(ctx context.Context, ln net.Listener) (*exe.Cm
+		fmt.Print("\033[1A\033[0K")
+	case "linux":
+		fmt.Print("\033[1A\033[0K")
+	default:
+		console.Error("Can't detect your operating system")
+	}
+	console.Info(msg)
+}
+
 func (c *Command) startApp(ctx context.Context, hotServer *hot.Server) error {
 	listener, err := socket.Load(c.Port)
 	if err != nil {
 		return err
 	}
+
+	// Init reload counter variable
+	totalReload := 0
+
 	// Compile and start the project
 	process, err := c.compileAndStart(ctx, listener)
 	if err != nil {
 		// TODO: de-duplicate with the watcher above
 		console.Error(err.Error())
 		if err := watcher.Watch(ctx, ".", func(path string) error {
+			totalReload++                       // Increase reload count
+			startTime := time.Now()             // Start timer
+			replacePreviousLine("Reloading...") // Handle reloading message
+
 			process, err = c.compileAndStart(ctx, listener)
 			if err != nil {
 				console.Error(err.Error())
+				console.Error("") // Avoid error messages getting overridden
+				totalReload = 0   // Reset reload counter
 				return nil
 			}
-			// TODO: host should be dynamic
-			console.Info("Ready on http://127.0.0.1" + c.Port)
+
+			// Time elapsed response
+			timeElapsed := time.Since(startTime).Milliseconds()
+			replacePreviousLine(fmt.Sprintf("Ready in %dms (x%d)", timeElapsed, totalReload))
 			return watcher.Stop
 		}); err != nil {
 			return err
 		}
 	}
 	defer process.Close()
+
 	// Start watching
 	if err := watcher.Watch(ctx, ".", func(path string) error {
 		switch filepath.Ext(path) {
 		// Re-compile the app and restart the Go server
 		case ".go":
+			totalReload++                       // Increase reload count
+			startTime := time.Now()             // Start timer
+			replacePreviousLine("Reloading...") // Handle reloading message
+
 			// Trigger a reload if there's a hot reload server configured
 			if hotServer != nil {
 				// Exclamation point just means full page reload
 				hotServer.Reload("!")
 			}
 			if err := process.Close(); err != nil {
 				console.Error(err.Error())
+				console.Error("") // Avoid error messages getting overridden
+				totalReload = 0   // Reset reload counter
 				return nil
 			}
 			app, err := c.Project.Compile(ctx, c.Flag)
 			if err != nil {
 				console.Error(err.Error())
+				console.Error("") // Avoid error messages getting overridden
+				totalReload = 0   // Reset reload counter
 				return nil
 			}
 			process, err = app.Start(ctx, listener)
 			if err != nil {
 				console.Error(err.Error())
+				console.Error("") // Avoid error messages getting overridden
+				totalReload = 0   // Reset reload counter
 				return nil
 			}
-			// TODO: host should be dynamic
-			console.Info("Ready on http://127.0.0.1" + c.Port)
+
+			// Time elapsed response
+			timeElapsed := time.Since(startTime).Milliseconds()
+			replacePreviousLine(fmt.Sprintf("Ready in %dms (x%d)", timeElapsed, totalReload))
 			return nil
 		// Hot reload the page
 		default:
```

Comment:
The reloading message should also be shown for non-Go files (e.g. Svelte files) in the `default` case.